pages/_app.tsx

```diff
@@ -7,60 +7,56 @@ import { DefaultSeo } from 'next-seo'
 import data from '../content/siteConfig.json'
 import TagManager from 'react-gtm-module'
 import { GlobalStyle } from '../components/styles/GlobalStyle'
-import { EditModeContext } from '../utils/editContext'
 
 const MainLayout = ({ Component, pageProps }) => {
-  const [isEditMode, setIsEditMode] = useState(false)
-
   const tinaConfig = {
     cms: {
       apis: {
         git: new GitClient('http://localhost:3000/___tina'),
       },
     },
     sidebar: {
-      hidden: !isEditMode,
+      // editMode initially set here
+      hidden: process.env.NODE_ENV === 'production' && true,
```

Comment:
I assume `&& true` is here to make it a boolean That's a fine approach, here's another:

```suggestion
      hidden: !!(process.env.NODE_ENV === 'production'),
```